docs/cli.md

```diff
@@ -0,0 +1,667 @@
+    "validatorAnnounce": "0x2fa5F5C96419C222cDbCeC797D696e6cE428A7A9",
+    "routingIsmFactory": "0x8358D8291e3bEDb04804975eEa0fe9fe0fAfB147",
+    "testRecipient": "0x4E1c88DD261BEe2941e6c1814597e30F53330428",
+    "testTokenRecipient": "0x5060eCD5dFAD300A90592C04e504600A7cdcF70b",
+    "pausableHook": "0x7556a0E61d577D921Cba8Fca0d7D6299d36E607E",
+    "fallbackRoutingHook": "0xD1E267d2d7876e97E217BfE61c34AB50FEF52807",
+    "interchainSecurityModule": "0xDEed16fe4b1c9b2a93483EDFf34C77A9b57D31Ff"
+  },
+  "inevm": {
+    "merkleRootMultisigIsmFactory": "0x2C1FAbEcd7bFBdEBF27CcdB67baADB38b6Df90fC",
+    "messageIdMultisigIsmFactory": "0x8b83fefd896fAa52057798f6426E9f0B080FCCcE",
+    "aggregationIsmFactory": "0x8F7454AC98228f3504Bb91eA3D8Adafe6406110A",
+    "aggregationHookFactory": "0xEb9FcFDC9EfDC17c1EC5E1dc085B98485da213D6",
+    "routingIsmFactory": "0x1052eF3419f26Bec74Ed7CEf4a4FA6812Bc09908",
+    "domainRoutingIsm": "0xBD70Ea9D599a0FC8158B026797177773C3445730",
+    "proxyAdmin": "0x0761b0827849abbf7b0cC09CE14e1C93D87f5004",
+    "storageGasOracle": "0x6119E37Bd66406A1Db74920aC79C15fB8411Ba76",
+    "interchainGasPaymaster": "0x19dc38aeae620380430C200a6E990D5Af5480117",
+    "merkleTreeHook": "0x0972954923a1e2b2aAb04Fa0c4a0797e5989Cd65",
+    "aggregationHook": "0xe0dDb5dE7D52918237cC1Ae131F29dcAbcb0F62B",
+    "protocolFee": "0x0D63128D887159d63De29497dfa45AFc7C699AE4",
+    "mailbox": "0x2f2aFaE1139Ce54feFC03593FeE8AB2aDF4a85A7",
+    "validatorAnnounce": "0x15ab173bDB6832f9b64276bA128659b0eD77730B",
+    "interchainSecurityModule": "0x3052aD50De54aAAc5D364d80bBE681d29e924597",
+    "pausableIsm": "0x6Fae4D9935E2fcb11fC79a64e917fb2BF14DaFaa",
+    "staticAggregationIsm": "0x3052aD50De54aAAc5D364d80bBE681d29e924597",
+    "pausableHook": "0xBDa330Ea8F3005C421C8088e638fBB64fA71b9e0"
+  },
+  "viction": {
+    "merkleRootMultisigIsmFactory": "0x2C1FAbEcd7bFBdEBF27CcdB67baADB38b6Df90fC",
+    "messageIdMultisigIsmFactory": "0x8b83fefd896fAa52057798f6426E9f0B080FCCcE",
+    "aggregationIsmFactory": "0x8F7454AC98228f3504Bb91eA3D8Adafe6406110A",
+    "aggregationHookFactory": "0xEb9FcFDC9EfDC17c1EC5E1dc085B98485da213D6",
+    "routingIsmFactory": "0x1052eF3419f26Bec74Ed7CEf4a4FA6812Bc09908",
+    "mailbox": "0x2f2aFaE1139Ce54feFC03593FeE8AB2aDF4a85A7",
+    "proxyAdmin": "0x0761b0827849abbf7b0cC09CE14e1C93D87f5004",
+    "validatorAnnounce": "0x2fa5F5C96419C222cDbCeC797D696e6cE428A7A9",
+    "interchainSecurityModule": "0xBD70Ea9D599a0FC8158B026797177773C3445730",
+    "merkleTreeHook": "0x149db7afD694722747035d5AEC7007ccb6F8f112",
+    "storageGasOracle": "0x19dc38aeae620380430C200a6E990D5Af5480117",
+    "interchainGasPaymaster": "0x0D63128D887159d63De29497dfa45AFc7C699AE4",
+    "protocolFee": "0xd83A4F747fE80Ed98839e05079B1B7Fe037b1638",
+    "testRecipient": "0x17E216fBb22dF4ef8A6640ae9Cb147C92710ac84",
+    "testTokenRecipient": "0xe042D1fbDf59828dd16b9649Ede7abFc856F7a6c"
+  }
+}
```

Comment:
As this is just an example, there is no need to include the entire output - simplify it to show an example for one chain and then let the rest be implied

```suggestion
{
  "sepolia": {
    "merkleRootMultisigIsmFactory": "0x0a71AcC99967829eE305a285750017C4916Ca269",
    "messageIdMultisigIsmFactory": "0xFEb9585b2f948c1eD74034205a7439261a9d27DD",
    "aggregationIsmFactory": "0xC83e12EF2627ACE445C298e6eC418684918a6002",
    "aggregationHookFactory": "0x160C28C92cA453570aD7C031972b58d5Dd128F72",
    "proxyAdmin": "0x97Bbc6bBaFa5Ce3b2FA966c121Af63bD09e940f8",
    "storageGasOracle": "0x71775B071F77F1ce52Ece810ce084451a3045FFe",
    "interchainGasPaymaster": "0x6f2756380FD49228ae25Aa7F2817993cB74Ecc56",
    "aggregationHook": "0xe3147d5618f5e2e100690B50ec923009a4cde14A",
    "protocolFee": "0x13AC3349Cb159fE86A22cf42DdA803D9f7309DB5",
    "mailbox": "0xfFAEF09B3cd11D9b20d1a19bECca54EEC2884766",
    "merkleTreeHook": "0x4917a9746A7B6E0A57159cCb7F5a6744247f2d0d",
    "validatorAnnounce": "0xE6105C59480a1B7DD3E4f28153aFdbE12F4CfCD9",
    "fallbackRoutingHook": "0x17Dc724B7a2F09141C13b8AC33B396073785c2BC",
    "testRecipient": "0xeDc1A3EDf87187085A3ABb7A9a65E1e7aE370C07",
    "testTokenRecipient": "0x031AD9c560D37baC7d6Bd2d27A2443bAfd10101A",
    "routingIsmFactory": "0x3F100cBBE5FD5466BdB4B3a15Ac226957e7965Ad"
  },
  "alfajores": {
    "merkleRootMultisigIsmFactory": "0xa9C7e306C0941896CA1fd528aA59089571D8D67E",
    ...
    "routingIsmFactory": "0x30d9A03762431F8A917a0C469E7A62Bf55092Ca6"
  },
  ...
}
```